src/gui/synthesispopover.h

```diff
@@ -87,7 +89,7 @@ class SynthesisPopover : public QDialog {
         void onRefreshErrorList(int driveDbId);
         void onAppVersionLocked(bool currentVersionLocked);
         void onRefreshStatusNeeded();
-        void onUpdateAvailabalityChange();
+        void onUpdateAvailabalityChange(UpdateState updateState);
```

Comment:
It is const on cpp side, not relevant in header. And the reference is not needed since it is just an enum value, there is no gain to pass it as reference.